styles.css

```diff
@@ -451,3 +443,286 @@ form {
+  }
+
+  .language-list ul li {
+    flex-direction: column;
+    max-width: 122px;
+    max-height: 120px;
+    padding: 2% 7% 25% 3%;
+  }
+
+  #third-page ul {
+    margin: 0 auto 32px 48px;
+  }
+
+  #header .nav_icon ul {
+    display: flex;
+    list-style: none;
+    margin-right: 154px;
+  }
+
+  #header .nav_icon ul li {
+    margin-right: 32px;
+  }
+
+  #header .nav_icon ul li:last-of-type {
+    margin-right: 0;
+  }
+
+  #header .nav_icon a {
+    display: none;
+  }
+
+  #first_page {
+    background-image: url(images/Header\ bg@2x.png);
+    background-size: cover;
+  }
+
+  .cover {
+    background-image: none;
+  }
+
+  .title h1 {
+    margin-left: 0;
+    padding: 0;
+  }
+
+  #first_page h1 {
+    margin-left: 260px;
+    padding-top: 281px;
+    margin-bottom: 12px;
+  }
+
+  .upper-part p {
+    margin-left: 0;
+    margin-top: 12px;
+    max-width: 447px;
+  }
+
+  .contact p {
+    margin-left: 436px;
+    margin-right: 434px;
+    margin-top: 12px;
+    margin-bottom: 12px;
+  }
+
+  #first_page p {
+    margin-left: 260px;
+    margin-bottom: 12px;
+  }
+
+  #first_page .connect {
+    margin-left: 260px;
+  }
+
+  #first_page .links ul {
+    margin-left: 260px;
+    padding: 0;
+    padding-bottom: 178px;
+  }
+
+  /* page two */
+
+  #mid {
+    display: grid;
+    grid-template-columns: 142px 1fr 142px;
+  }
+
+  .left-block {
+    grid-column: 2;
+    flex-direction: row;
+  }
+
+  #grid-eight {
+    align-items: left;
+  }
+
+  #second-page .left {
+    margin-top: 48px;
+    margin-left: 46px;
+  }
+
+  #grid-two .primary-text {
+    margin-left: 0;
+    padding-left: 0;
+  }
+
+  #grid-two .tags {
+    margin-left: 0;
+  }
+
+  .frame-2 {
+    margin: 0;
+    padding-bottom: 0 0 0 20px;
+  }
+
+  #grid-two p {
+    margin-left: 46px;
+    max-width: 419px;
+  }
+
+  .left .action {
+    margin-left: 0;
+  }
+
+  .for-small-size {
+    display: none;
+  }
+
+  /* page three */
+
+  #third-page {
+    flex-direction: row-reverse;
+  }
+
+  .right {
+    flex: 1;
+  }
+
+  #third-page h1 {
+    margin: 48px 46px 20px 48px;
+  }
+
+  #third-page .frame-2 {
+    margin: 0;
+    padding: 0;
+  }
+
+  #third-page p {
+    margin: 0 auto 32px 48px;
+    max-width: 498px;
+  }
+
+  .frame-2 .action {
+    margin-left: 60px;
+  }
+
+  #third-page .action {
+    margin: 0 auto 32px 48px;
+  }
+
+  /* page four */
+
+  #fourth-page .left {
+    padding-top: 48px;
+    padding-left: 46px;
+  }
+
+  .left .for-small-size {
+    display: none;
+  }
+
+  #grid-six p {
+    padding-bottom: 26px;
+  }
+
+  /* page five */
+
+  #page-five {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  #page-five .right {
+    margin-left: 48px;
+    margin-top: 48px;
+  }
+
+  #page-five p {
+    padding-bottom: 26px;
+  }
+
+  .right .for-small-size {
+    display: none;
+  }
+
+  /* About page */
+
+  .about-block {
+    display: flex;
+    flex-direction: row;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
```

Comment:
- [ ] Kindly increase the margin between your about-me title and description to match your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194627325-425639bc-3a09-4379-bac8-91bc5f417b7f.png)  |![image](https://user-images.githubusercontent.com/63473584/194627444-0f0d4450-7222-416f-8047-984bd92717ab.png) |